core/admin/mailu/ui/templates/sidebar.html

```diff
@@ -5,7 +5,7 @@
       <div class="div-circle elevation-2"><i class="fa fa-user text-lg text-dark"></i></div>
     </div>
     <div class="info">
-      <a href="{{ url_for('.user_settings') }}" class="d-block">{{ current_user }}</a>
+      <a href="{{ url_for('.user_settings') }}" class="d-block">{{ current_user.displayed_name or current_user }}</a>
```

Comment:
Sorry, but I think this also should be escaped.
I didn't test this but Jinja has an escape filter called "e", so this should work:
`current_user.displayed_name|e or current_user|e`
Could you add a title attribute to the div (class user-panel) above containing the email-address?
`<div class="user-panel mt-3 pb-3 mb-3 d-flex" title="{{ current_user|e }}">`

Reply:
I thnk jinja autoescape inputs if isn't disabled : https://flask.palletsprojects.com/en/1.1.x/templating/#controlling-autoescaping

I add title attribute on user-panel div.